src/commonMain/kotlin/exchange.dydx.abacus/calculator/TradeInputCalculator.kt

```diff
@@ -26,6 +26,14 @@ enum class TradeCalculation(val rawValue: String) {
     }
 }
 
+object SlippageConstants {
+    internal const val MARKET_ORDER_MAX_SLIPPAGE = 0.05
```

Comment:
nit, can do `internal object` and drop all the `internal` from the fields inside

Reply:
so much kotlin syntax im learning ty!!